### PR TITLE
Bugfix: Boundary of some elements cease to exist while resizing

### PR DESCRIPTION
**Pull request by minrows**
…ments

### Checklist
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes

### Motivation and Context
To specify the named bounds so that the issue with vanishing element boundary, while resizing is resolved.
Issue Number: https://github.com/ls1intum/Apollon/issues/208

### Steps for Testing
1. Open Apollon
2. Select Activity Diagram from Diagram Type
3. Drag Activity Action node to the canvas and start reducing its width/height
4. Observe that the element now no longer vanishes while its side is reduced as it used to (see: https://github.com/ls1intum/Apollon/issues/208 ).

### Screenshots
screen-capture-_7_

### CODE REVIEW

File: src/main/packages/flowchart/flowchart-decision/flowchart-decision.tsx [hunk 1]
```diff
@@ -2,12 +2,15 @@ import { FlowchartElementType } from '..';
...
 import { UMLElement } from '../../../services/uml-element/uml-element';
+import { calculateNameBounds } from '../../../utils/name-bounds';
 import { UMLElementType } from '../../uml-element-type';
 
 export class FlowchartDecision extends UMLElement {
   type: UMLElementType = FlowchartElementType.FlowchartDecision;
 
   render(canvas: ILayer): ILayoutable[] {
+    const namedBounds = calculateNameBounds(this, canvas);
+    this.bounds = namedBounds;
```
Comment (martindevtum) [inline src/main/packages/flowchart/flowchart-decision/flowchart-decision.tsx:13]: I would personally skip the creation of the constant and just assign to this.bounds directly, as we are not using namedBounds anywhere else

File: src/main/utils/name-bounds.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { ILayer } from '../services/layouter/layer';
+import { IBoundary, computeBoundingBoxForElements } from '../utils/geometry/boundary';
+import { Text } from '../utils/svg/text';
+
+export function calculateNameBounds(element: any, layer: ILayer) {
```
Comment (martindevtum) [inline src/main/utils/name-bounds.ts:5]: shouldn't we add the return type IBoundary here?

**Review by martindevtum — CHANGES_REQUESTED** [review]
I would also like it if you would use the newly created name bounds function to already existing usages of nameBounds. 

It is for example used in uml-package.ts, please calculate the nameBounds here with our new function. 
This might be the only occurence, but please just. double check it again.


Other than that, it is working nicely.

**Comment by bassner** [comment]
I tested this locally and it seems to work well, except for packages (class diagram), which can still cut off their text content vertically:
grafik
But I'm not 100% sure if that was even the point of this PR or if it was just about keep the correct width on certain objects?

**Comment by minrows** [comment]
> 

Done. Minimum height is now calculated based on the name as well.

**Review by bassner — APPROVED** [review]
LGTM!

**Comment by martindevtum** [comment]
looking good

**Review by martindevtum — APPROVED** [review]

**Pull request merged**